src/shared/directives/conditional-show/domain/conditional-show.model.spec.ts

```diff
@@ -607,4 +607,102 @@ describe('conditional-show', () => {
         'ComplexField1.AddressLine1')).toBe('ComplexField1.AddressLine1.field1="test" AND ComplexField1.AddressLine1.field2CONTAINS"s1"');
     });
   });
+  describe('NOT EQUALS', () => {
+    it('Scenario1 show: comparator match with specific value', () => {
+      let sc = new ShowCondition('field!="MOJ"');
+      let fields = {
+        field: 'Reform'
+      };
+      let matched = sc.match(fields);
+      expect(matched).toBe(true);
+    });
+    it('Scenario2 show: is not blank', () => {
+      let sc = new ShowCondition('field!=""');
+      let fields = {
+        field: 'MOJ'
+      };
+      let matched = sc.match(fields);
+      expect(matched).toBe(true);
+    });
+    it('Scenario2 hide: is not blank', () => {
```

Comment:
can we have a test 'is not blank multiple spaces' i.e.
`'field!="     "'`

Reply:
DONE